cmd/sonobuoy/app/status.go

```diff
@@ -19,28 +19,61 @@ package app
 import (
 	"os"
 
-	ops "github.com/heptio/sonobuoy/cmd/sonobuoy/app/operations"
-	"github.com/heptio/sonobuoy/pkg/errlog"
+	"github.com/olekukonko/tablewriter"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/heptio/sonobuoy/cmd/sonobuoy/app/args"
```

Comment:
This shouldn't be pulling in args, see https://github.com/heptio/sonobuoy/pull/264

Reply:
I thought `cmd` was the right place for args and `operations` was the wrong place for args.

Reply:
yeah, my mistake.